x-pack/plugins/enterprise_search/public/applications/app_search/components/curations/views/curations_settings/curations_settings.test.tsx

```diff
@@ -227,7 +229,7 @@ describe('CurationsSettings', () => {
       const wrapper = shallow(<CurationsSettings />);
       expect(wrapper.is(DataPanel)).toBe(true);
       expect(wrapper.prop('action').props.to).toEqual('/app/management/stack/license_management');
-      expect(wrapper.find(EuiButtonEmpty).prop('href')).toEqual('/license-management.html');
+      expect(wrapper.find(EuiButtonEmpty).prop('href')).toEqual(`${docLinks.licenseManagement}`);
```

Comment:
```suggestion
      expect(wrapper.find(EuiButtonEmpty).prop('href')).toEqual(docLinks.licenseManagement);
```
No need for a template literal here